packages/hardhat-core/src/internal/hardhat-network/jsonrpc/server.ts

```diff
@@ -72,7 +72,7 @@ export class JsonRpcServer implements IJsonRpcServer {
 
   public close = async () => {
     await Promise.all([
-      new Promise((resolve, reject) => {
+      new Promise<void>((resolve, reject) => {
```

Comment:
This doesn't seem necessary?

Reply:
If not changed eslint throws an error

Reply:
Really? That's weird, I changed it and it didn't complain :/ Maybe it's a setup thing?

Reply:
Highly likely, should I discard this change? I feel like it does no harm.

Reply:
Yeah, I'd rather not have unnecessary type annotations.

Reply:
Ok removed